course/templates/course/gradebook-by-opp.html

```diff
@@ -109,12 +111,17 @@ <h3 class="start-well-title">{% trans "Modify many sessions at once" %}</h3>
           <a href="{% url "relate-view_single_grade" course.identifier participation.id opportunity.id %}"><span class="sensitive">{{ participation.user.username }}</span></a>
         </td>
         <td class="datacol">
-            <span class="sensitive">
-                {{ participation.user.get_full_name }}
```

Comment:
I am wondering why the user.get_full_name here is reverted back to 2 columns ;-(